docs/analytics.md

```diff
@@ -0,0 +1,132 @@
+
+<i>Last updated: Feb 25, 2019</i>
+
+<br />
+
+# Firefox Monitor Analytics Plan
+
+This is the Analytics plan for Firefox Monitor. It documents our use of Google Analytics and what we do with the information we collect.
+
+<br>
+
+## Analysis
+**Firefox Monitor uses Google Analytics to collect and organize data. We do this to get a better understanding of what is working, and where we still have work to do.**
+
+
+**Captured data also helps provide answers to the following questions:**
+
+
+**Demographic:**
+
+>From which country does the majority of our traffic originate?
+
+>In which language does the majority of our traffic view the site?
+
+>Which browsers are most commonly used to access Firefox Monitor?
+
+>Which devices are most commonly used to access Firefox Monitor?
+
+<br>
+  
+**User Behavior:**
+
+>Do users scan their email?
+
+>Do users scan multiple email addresses?
+
+>What percentage of users sign up receive alerts from Firefox Monitor?
+
+>Are users more likely to sign up for alerts before or after scanning their email?
+
+
+<br />
+
+## Collection
+
+Events are reported using the [Google Analytics Measurement Protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1/).
+
+We collect data for the following events:
+
+- When the page loads (or reloads).
+`hitType` : pageView
+`page` : location.pathName
+<br>
+
+#### Forms:<br><br> `Email Scan Form` &nbsp; `Scan Another Email Form` &nbsp; `Sign Up Form` &nbsp;
+
+- When a form appears on the page.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : View
+`eventLabel` : Page location ID.
+
+
+- When a user focuses inside a form input.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Engage
+`eventLabel` : Page location ID.
+
+
+- When a user submits a form.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Submit
+`eventLabel` : Page location ID.
+
+
+- When a user submits an invalid email.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Failure
+`eventLabel` : Page location ID.
+
+
+- When a user submits a valid email.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : Success
+`eventLabel` : Page location ID.
+<br>
+
+
+#### Buttons:<br><br> `Download Firefox Button`&nbsp;&nbsp;`Sign Up Button`&nbsp;&nbsp; `See Full Report Button`<br><br> `Sign up to Check Button` &nbsp;`Show All Breaches Button`
```

Comment:
Same as above... Probably switch to an `<h3>` level heading (`###`) and then find an alternate way of displaying the list of buttons. Also, because some of these are using one `&nbsp;` and some two (and one may have 3) and they span multiple lines.

### Buttons:

#### `Download Firefox Button` &mdash; `Sign Up Button` &mdash; `See Full Report Button` &mdash; `Sign up to Check Button` &mdash; `Show All Breaches Button`

Or, a list:

- **`Download Firefox Button`**
- **`Sign Up Button`**
- **`See Full Report Button`**
- **`Sign up to Check Button`**
- **`Show All Breaches Button`**

Or, a table:

| `Download Firefox Button` | `Sign Up Button` | `See Full Report Button` | `Sign up to Check Button` | `Show All Breaches Button` |
|---------------------------|------------------|--------------------------|----------------------------|----------------------------|